packages/relay/src/lib/eth.ts

```diff
@@ -669,12 +669,26 @@ export class EthImpl implements Eth {
                   currentTimestamp = mirrorAccount.balance.timestamp;
                 }
 
-                let transactionsInTimeWindow = await this.mirrorNodeClient.getTransactionsForAccount(
-                  mirrorAccount.account,
-                  block.timestamp.to,
-                  currentTimestamp,
-                  requestId
-                );
+                // Need to check if there are any transactions before the block.timestamp.to in the current account set returned from the inital
+                // call to getAccountPageLimit.  If there are we may need to paginate.
+                let lastTransactionOnPageTimestamp;
+                if(mirrorAccount.links.next !== null) {
+                  // Get the end of the page of transactions timestamp
+                  lastTransactionOnPageTimestamp = mirrorAccount.links.next.substring(mirrorAccount.links.next.indexOf("gte:") + 4, mirrorAccount.links.next.indexOf("&timestamp"));
```

Comment:
Q: Is this safe way to parse the string out?
What if the params order are changed on the mirror node side, we'd be broken even though the functionality didn't change.
Would it be better to split the params on `&`, take the time stamp params and split that on appropriately?

Reply:
Updated to safer parsing with logging of unintended results.